components/CanvasOne.js

```diff
@@ -0,0 +1,54 @@
+import { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+
+export default function CanvasOne({
+  canvasOneParameter,
```

Comment:
Same here for this parameter name that is related to your component name.

Reply:
Deleted "ONE"! 😩